frontend/src/styling/styles.ts

```diff
@@ -6,11 +6,15 @@ export const Paragraph = styled.p`
   ${tw`text-base font-bold m-auto`}
 `
 
-export const GeyserFirstOverlay = styled.div`
+const Overlay = styled.div`
   ${tw`shadow-all w-full rounded-lg my-2`}
   ${tw`sm:my-4`}
 `
 
+export const GeyserFirstOverlay = styled(Overlay)``
+
+export const VaultFirstOverlay = styled(Overlay)``
+
```

Comment:
actually since there is no custom styling, we can just use the `Overlay` you created and purge these two

Reply:
fixed